### DISCUSSION TITLE
Project roadmap

### DISCUSSION THREAD

**Discussion by cometkim** [daangn/seed-design#37] [Announcements]
### Problem

- 당근마켓은 다양한 목적조직으로 고도로 분산되어서 일합니다.
- 디자인 라이브러리의 중앙화는 조직이 일하는 방식과 맞지 않고 커다란 단일 병목 지점이 될 가능성이 커보입니다.
- 당근마켓은 웹에 국한되지 않고 다양한 플랫폼 기술을 활용합니다.
- 당근마켓은 UI 개발과정에서 각자 다양한 도구를 활용합니다.

### Non-goals

- karrot-ui가 목표하는 바는 중앙화를 통해 UI 개발방식을 획일화해서 생산성을 끌어올리고자 함이 아닙니다.

### Goals

- karrot-ui는 UI 에 최소한의 일관성을 부여하고 분산 조직의 상호의존적 커뮤니케이션에서 생기는 언어 차이를 해소하려는 시도입니다.
- karrot-ui는 당근마켓 디자인 시스템이 어떤식으로 코드로 적용될 수 있는지 개발자들에게 충분한 예제를 제공합니다.
- karrot-ui 에서 제공하는 라이브러리는 "레퍼런스 구현"이며 다른 라이브러리 구성을 제한하지 않습니다.
- karrot-ui는 당근마켓 디자인 시스템에 따라 새로운 디자인 라이브러리를 만들고자 하는 개발자에게 적절한 파운데이션과 빌딩 블럭들을 제공합니다.

### Ideas

- 디자인 시스템은 제품이고, 다른 조직에 접근하는 방식은 명령 하달이 아니라 세일즈에 가깝습니다.
- karrot-ui 플랫폼이고 플랫폼의 사용의도에 미치는 영향도는 karrot-ui 자체보다 그 주변에 형성되는 네트워크가 더 큽니다
- karrot-ui는 디자인 시스템에 대한 네트워크 이펙트를 발생시키는데 주력해야합니다. 주요한 수단은 문제를 도와줄 커뮤니티와 좋은 가이드입니다.

### Governance model

TBD

### To Dos

TBD

**Comment by 0261** [comment]
이런데 숨켜놓으니까 아무도 의견을 안달지..

**Reply by cometkim** [reply]
에... 홍보 여러번 했음 -_-
